pkgs/by-name/ca/catch2/package.nix

```
@@ -3,30 +3,75 @@
   stdenv,
   fetchFromGitHub,
   cmake,
+  python3,
+  spdlog,
 }:
 
 stdenv.mkDerivation rec {
   pname = "catch2";
-  version = "2.13.10";
+  version = "3.11.0";
```

```suggestion
  version = "3.12.0";
```
https://github.com/NixOS/nixpkgs/pull/474924